Gemfile

```diff
@@ -1,8 +0,0 @@
-source 'https://rubygems.org'
-
-group :jekyll_plugins do
-  gem 'github-pages'
-  gem 'jekyll-paginate'
-  gem 'jekyll-assets'
-  gem 'jekyll-contentblocks'
-end
```

Comment:
Not sure if this file is needed since we're killing Jekyll.

Reply:
the replacement should auto-generate a new Gemfile with the Gems needed for that project. you're correct that this is only needed for Jekyll